api/apps/articles/model.coffee

```diff
@@ -114,6 +114,8 @@ inputSchema = (->
     ]).default([])
   email_metadata: @object().keys
     image_url: @string().allow('',null)
+    small_image_url: @string().allow('',null)
+    large_image_url: @string().allow('',null)
```

Comment:
Wondering now if it's better to just construct the `small_image_url` and `large_image_url` on Force. We need these for `<meta sailthru.image.thumb>` and `<meta sailthru.image.full>` tags. Reason is, if we want ST to spider old content, they might not have these fields attached.